pkg/master/master.go

```diff
@@ -669,28 +661,36 @@ func (m *Master) thirdpartyapi(group, kind, version string) *apiserver.APIGroupV
 	}
 }
 
+func (m *Master) GetRESTOptionsOrDie(c *Config, resource unversioned.GroupResource) generic.RESTOptions {
+	storage, err := c.StorageDestinations.Get(resource)
+	if err != nil {
+		glog.Fatal(err.Error())
```

Comment:
Add more info to this error, so that someone reading it might have a chance of figuring out what they can do to fix the situation.